backend/business-partner-agent-core/src/main/java/org/hyperledger/bpa/core/WebhookEvent.java

```diff
@@ -17,12 +17,11 @@
  */
```

Comment:
License header is not up to date

Reply:
can be reverted anyway, as this does nothing

Reply:
Yes the "optimize imports"  process rearranged imports alphabetically, so this change will be reverted.